project/Dependencies.scala

```diff
@@ -0,0 +1,50 @@
+import sbt._
+
+object Dependencies {
+
+  val KafkaVersion = "2.8.0"
+
+  object Akka {
+    val akka          = "2.6.16"
```

Comment:
minor but `version` seems a more appropriate name?

Reply:
👍  renamed to `version` and made private